graphql2/schema.graphql

```diff
@@ -38,6 +38,12 @@ type Query {
   # Returns a single schedule with the given ID.
   schedule(id: ID!): Schedule
 
+  # Returns the public information of a calendar subscription
+  calendarSubscription(id: ID!): CalendarSubscription
+
+  # Returns a paginated list of calendar subscriptions
+  calendarSubscriptions(input: CalendarSubscriptionSearchOptions): CalendarSubscriptionConnection!
```

Comment:
This should exist on the `User` struct as previously discussed in the proposal: https://github.com/target/goalert/compare/calendar-subscription-api#diff-04e3af9ffc876a383b6f192a8ff8450dR841